drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -212,9 +212,14 @@ function DragAndDropTemplates(configuration) {
             gettext('Items placed here: ') + items_in_zone.map(function (item) { return item.displayName; }).join(", ")
           );
         }
+
+        /* Force zone alignment if showing answer. */
+        if (ctx.show_answer && zone.align === 'none') {
+          zone.align = 'center'
+        }
+
         if (zone.align !== 'none') {
           item_wrapper += '.item-align.item-align-' + zone.align;
-          //items_in_zone = $.grep(ctx.items, is_item_in_zone);
```

Comment:
To answer your question, because it's handled above, on line 200, and the author of this changeset probably forgot the commented line in.

On the merits of the implemented logic, the point of this `if/else` block is to avoid rendering any unaligned items in the zone template: these are rendered elsewhere.